### PR TITLE
[15.0][MIG]project_role: Migration to v15

### PR DESCRIPTION
**Pull request by bizzappdev**

### CODE REVIEW

**Comment by bizzappdev** [comment]
> Hey @bizzappdev, thank you for your Pull Request.
> 
> It looks like some users haven't signed our **C**ontributor **L**icense **A**greement, yet. You can read and sign our full Contributor License Agreement here: http://odoo-community.org/page/cla Here is a list of the users:
> 
> * alfadil [alfadil.tabar@gamil.com](mailto:alfadil.tabar@gamil.com) (no github login found)
> 
> Appreciation of efforts, OCA CLAbot

that is from commit history not related to our contribution.

**Review by ajaniszewska-dev — APPROVED** [review]

**Comment by rafaelbn** [comment]
Hello, @alexey-pelykh ! are you interested in this one? 😄

File: project_role/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Copyright 2018-2019 Brainbean Apps (https://brainbeanapps.com)
+# Copyright 2020 CorporateHub (https://corporatehub.eu)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Project Roles",
+    "version": "15.0.0.0.1",
```
Comment (rafaelbn) [inline project_role/__manifest__.py]: 15.0.1.0.0  @bizzappdev

**Comment by alexey-pelykh** [comment]
:) This migration seems straightforward :)

**Review by rafaelbn — APPROVED** [review]

**Review by rafaelbn — COMMENTED** [review]
Please change https://github.com/OCA/project/pull/896#pullrequestreview-932372699

**Comment by bizzappdev** [comment]
> Please change [#896 (review)](https://github.com/OCA/project/pull/896#pullrequestreview-932372699)

change is done

File: project_role/models/project_assignment.py [hunk 2]
```diff
@@ -79,22 +79,23 @@ class ProjectAssignment(models.Model):
     def _compute_name(self):
         for assignment in self:
             if assignment.project_id:
-                assignment.name = _("%s as %s on %s") % (
-                    assignment.user_id.name,
-                    assignment.role_id.name,
-                    assignment.project_id.name,
-                )
+                assignment.name = _(" %(user_id)r as %(role_id)r on %(project_id)r") % {
```
Comment (alexey-pelykh) [inline project_role/models/project_assignment.py:82]: Why space in front is needed?

**Comment by alexey-pelykh** [comment]
Haven't heard back in over 2 months, thus considering this PR abandoned. Superseded in #949

**Pull request closed without merging**